Maquetado/scss/styles.scss

```diff
@@ -1,3 +1,11 @@
-.title {
-  color: red;
+$text-big: 24px;
```

Comment:
size variables should be placed in a separate file called `sizes.scss` and imported in the `application.scss`